Reed/Challenges/25/index.css

```diff
@@ -0,0 +1,27 @@
+.button-container{
+    margin-top: 10rem;
+    display: flex;
+    justify-content: center; 
+    align-items: center;
+    background-color: #307fe2;
+    clip-path: polygon(7.5% 0, 100% 0%, 92.5% 100%, 0% 100%);
+    font-size: 1rem;
+    text-decoration: none;
+    width: 400px;
+}
+.button-container:visited { text-decoration: none; color:none; }
+.button-container:hover { text-decoration: none; color:none; }
+.button-container:focus { text-decoration: none; color:none; }
+
+.button-container:hover, a:active { 
+    text-decoration: none; 
+    color:black;
+    cursor: pointer;
+    filter: brightness(1.15); 
```

Comment:
I also haven't seen filter: brightness used before either.

Me thinking outloud: Do you think there are benefits to using filter over an explicit change in the background-color style? Would we want to update the use color that we use originally or is filter a better use?